src/sacn/receiver.c

```diff
@@ -131,7 +131,10 @@ static void process_receiver_sources(sacn_thread_id_t thread_id, SacnReceiver* r
                                      SourcesLostNotification* sources_lost);
 static bool check_source_timeouts(SacnTrackedSource* src, SacnSourceStatusLists* status_lists);
 static void update_source_status(SacnTrackedSource* src, SacnSourceStatusLists* status_lists);
-static void deliver_periodic_callbacks(const SourcesLostNotification* sources_lost_arr, size_t num_sources_lost);
+static void deliver_periodic_callbacks(const SourcesLostNotification* sources_lost_arr, size_t num_sources_lost,
+                                       const SamplingStartedNotification* sampling_started_arr,
+                                       size_t num_sampling_started, const SamplingEndedNotification* sampling_ended_arr,
+                                       size_t num_sampling_ended);
```

Comment:
There's enough of these that I think we might want to have a PeriodicCallbacks structure that we pass in.